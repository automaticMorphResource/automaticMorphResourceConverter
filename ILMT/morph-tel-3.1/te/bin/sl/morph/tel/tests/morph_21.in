1	koVvAlA	unk	
1	,	unk	
1	sIlsu	unk	
1	,	unk	
1	eVkidnA	unk	
1	,	unk	
1	kaMgArU	unk
1	..	unk	
1	waxiwara	unk	
1	jaMwujAlaM	unk	
1	hAyigA	unk	
1	,	unk	
1	surakRiwaMgA	unk	
1	ikkada	unk	
1	saMcariswU	unk	
1	uMtAyi	unk	
1	.	unk	
