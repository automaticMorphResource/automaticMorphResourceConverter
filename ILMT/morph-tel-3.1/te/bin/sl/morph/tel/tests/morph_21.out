1	koVvAlA	unk	<fs af='koVvAlA,unk,,,,,,'>
2	,	unk	<fs af='&comma;,punc,,,,,,'>
3	sIlsu	unk	<fs af='sIlsu,n,,sg,,d,0,0'>|<fs af='sIlsu,n,,sg,,o,ti,ti'>|<fs af='sIls,n,,sg,,d,0,0'>|<fs af='sIls,n,,sg,,o,ti,ti'>
4	,	unk	<fs af='&comma;,punc,,,,,,'>
5	eVkidnA	unk	<fs af='eVkidnA,n,,sg,,d,0,0'>|<fs af='eVkidnA,n,,sg,,o,ti,ti'>
6	,	unk	<fs af='&comma;,punc,,,,,,'>
7	kaMgArU	unk	<fs af='kaMgArU,n,,sg,,d,0,0'>|<fs af='kaMgArU,n,,sg,,o,ti,ti'>|<fs af='kaMgAru,n,,sg,,,0,0_V'>
8	..	unk	<fs af='&dot;,punc,,,,,,'>
9	waxiwara	unk	<fs af='waxiwara,adj,,,,,0,0_adj'>|<fs af='waxiwaraM,n,,sg,,o,ti,ti'>|<fs af='waxiwarudu,n,,sg,,o,ti,ti'>
10	jaMwujAlaM	unk	<fs af='jaMwujAlaM,n,,sg,,d,0,0'>
11	hAyigA	unk	<fs af='hAyigA,avy,,,,,0,0_avy'>|<fs af='hAyi,n,,sg,,,gA,gA'>
12	,	unk	<fs af='&comma;,punc,,,,,,'>
13	surakRiwaMgA	unk	<fs af='surakRiwaM,n,,sg,,,gA,gA'>
14	ikkada	unk	<fs af='ikkada,n,,sg,,d,0,0'>|<fs af='ikkada,n,,sg,,o,ti,ti'>
15	saMcariswU	unk	<fs af='saMcariMcu,v,any,any,any,,wU,wU'>
16	uMtAyi	unk	<fs af='uMdu,v,n,pl,3,,wA,wA'>
17	.	unk	<fs af='&dot;,punc,,,,,,'>
